ace

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env node --harmony_proxies
 'use strict'
-
+/*
+|--------------------------------------------------------------------------
+|   Running Console Commands
+|--------------------------------------------------------------------------
+|
+|  Here we invoke console commands registered under Ace store.
+|
+*/
 const kernel = require('./bootstrap/kernel')
-kernel()
+kernel()
```

Comment:
This file shouldn't be changed, also there's no new line at the end of the file.